### ISSUE TITLE
blog: 1 missing/broken image (migration error)

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov.gov#750]
https://digital.gov/2017/04/04/color-in-digital-design/ 

## it's done when …
- [x] image is uploaded and displays on pg

**Comment by ToniBonittoGSA** [comment]
@jeremyzilar, the image code currently has two errors in the filename:

> img src="https://s3.amazonaws.com/digitalgov/_legacy-img/2017/04/600-x-461-Boutet%5c_1708%5c_color_circles.jpg" alt="Claude Boutet’s 7-color and 12-color color wheels."

The `%5c` is ASCII code for backslash `\` 

In WordPress, the file was named, *600-x-461-Boutet_1708_color_circles.jpg*
somehow, during migration, a backslash was added twice; after Boutet, and after 1708.

Here is the [original image from Wikipedia](https://commons.wikimedia.org/wiki/File:Boutet_1708_color_circles.jpg) (a little bigger than 600)

![boutet_1708_color_circles](https://user-images.githubusercontent.com/19961574/45165436-d16aa880-b1c2-11e8-9ae5-46fcc2a1821f.jpg)

alt text: 
Claude Boutet’s 7-color and 12-color color wheels.

caption/attribution: 
By C. B. (probably Claude Boutet or it's editor, Christophe Ballard[1]) [Public domain], via Wikimedia Commons


Thx!

**Comment by jeremyzilar** [comment]
Here is what I found out.
The image path that is in there now is:
`https://s3.amazonaws.com/digitalgov/_legacy-img/2017/04/600-x-461-Boutet%5c_1708%5c_color_circles.jpg`

So what is wrong with that path? Why isn;t it working? And what happened to make it not work?
It looks like special chars in the migration were changed in that path.

Specifically, there is a `%5c` in the URL that should not be there. If I remove both of them, this is the path I get:
`https://s3.amazonaws.com/digitalgov/_legacy-img/2017/04/600-x-461-Boutet_1708_color_circles.jpg`

And it works! 🎉 

Now we should figure out if there are other broken images in the site that are also afflicted by the same bug. It should be easy to fix across the board.
